<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weather Judgment Hub</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="manual_tab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>781</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>en attente de DATA .</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Température :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="input_vent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pression atmosphérique :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="input_temp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="input_atmo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vittesse vent :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="api_tab">
     <attribute name="title">
      <string>API</string>
     </attribute>
     <widget class="QLineEdit" name="input_api_keys">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_ville">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Votre clée API</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_api">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Votre c'est API est </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Ville :</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="API_Start_B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Lancer l'appel API</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
